record.html

```diff
@@ -0,0 +1,88 @@
+
+      <div class="content-block">
+        <ol>
+          <li>Say your name, phone number, and that you’re a voter.</li>
+          <li>Explain that you’re calling to urge the Representative to become a leader for reform and to <span class="strong">support HR20</span>, <a href="http://www.repswith.us/reforms/government-by-the-people-act" target="_blank">The Government By The People Act.</a></li>
+          <li>Thank the Representative for listening to citizens instead of special interests.</li>
+          <li>When you finish your message, hang up or press # to review.</li>
+        </ol>
+
+        <small>Remember: be confident, speak slowly, and don’t forget to breathe.</small>
+      </div>
+
+      
+      <h2>Ready? Dial this number:</h2>
+      <div class="cta">
+        <span class="number">
+          540-409-4371
+        </span>
+      </div>
+
+      Note: your number will be on the caller ID when we deliver this message. 
+      <br>
+      Thank you for working to repair our republic. 
+    </div>
+  </div>
+  <br>
+
+  <div class="panel action-panel with-shield">
+    <div class="panel-icon">
+      <img src="/images/M_shield_callout_box.svg" class="img-responsive" alt="">
+      <div class="fade-line hidden-xs hidden-sm"></div>
+    </div>
+    <div class="panel-body">
+      <div class="row">
+        <h2>Spread the Word</h2>
+        <p>After you leave your message, invite your friends to leave their message for Congress</p>
+      </div>
+      <div class="row">
+        <div class="col-md-4">
+          <a href="http://www.facebook.com/sharer.php?u=https://mayday.us/record" target="_blank" class="btn btn-md-wide btn-share-center facebook">
+            <span class="text"><i class="fa fa-facebook"></i></span>
+          </a>
+        </div>
+        <div class="col-md-4">
+          <a href="https://twitter.com/home?status=Record%20a%20message%20that%20will%20be%20delivered%20to%20key%20Congress%20members%20to%20reform%20campaign%20funding.%20mayday.us%2Frecord%20%40maydayus%20%23Leader4Reform" target="_blank" class="btn btn-md-wide btn-share-center twitter">
+            <span class="text"><i class="fa fa-twitter"></i></span>
+          </a>
+        </div>
+        <div class="col-md-4">
+          <a href="mailto:?Subject=Send%20Congress%20a%20voice%20message&body=Hey%2C%20check%20it%20out!%20You%20can%20record%20a%20simple%20message%20that%20MAYDAY%20will%20deliver%20to%20key%20Congressional%20offices%20to%20reform%20the%20way%20campaigns%20are%20funded.%20It%20takes%201%20minute--thanks%20in%20advance!%20https%3A%2F%2Fmayday.us%2Frecord%2F" class="btn btn-md-wide btn-share-center">
+            <span class="text"><i class="fa fa-envelope-o"></i></span>
+          </a>
+        </div>
+      </div>
+    </div>
+  </div>
+
+{% endcapture %}
+
+{% include sub_layouts/default_layout.html %}
+
+<script>
```

Comment:
@Rio517 If theres a better place to put this please let me know. (I haven't finished cleaning up the rest of the JS yet)